### ISSUE TITLE
Node Status no longer showing

### ISSUE THREAD

**Issue by shawwellpete** [hassio-addons/addon-node-red#1213]
Hi Guys,

I assume this has happened after a recent update, I am running 10.2.0 on Home Assistant 2021.11.5. 

I no longer get any status updates on nodes with node red running.

I have the "show node status" box ticked. 

Any other suggestions?

![2021-12-13_11-59-16](https://user-images.githubusercontent.com/74064242/145808544-7eb22ff0-54a6-4f76-b68b-c19763c6e240.jpg)

**Comment by bobslaede** [comment]
Same here. Also, debug output is not working either. Do you have the same issue?

**Comment by shawwellpete** [comment]
> Same here. Also, debug output is not working either. Do you have the same issue?

I hadn't noticed that but yes, that too!

**Comment by bobslaede** [comment]
It sucks :( really difficult to debug stuff to get it working.

**Comment by shawwellpete** [comment]
I asked about it on the [Home Assistant face book group](https://www.facebook.com/groups/HomeAssistant/posts/3111748242429858/) and at least  one person hasn't got the issue.

I wonder what we have done to  cause it?

**Comment by bobslaede** [comment]
I have also found someone that has the same version of HA and same version of Node-red as me, who does not have the problem. Not sure what we have done wrong.

**Comment by shawwellpete** [comment]
Weirdly restarting nodered fixes it for a short while, but when you go out of node red back to supervisor the problem happens again.

**Comment by bobslaede** [comment]
That is pretty weird :/ also brings back status. Will keep an eye on it

**Comment by bobslaede** [comment]
With the latest update, it seems stable enough

**Comment by shawwellpete** [comment]
So you have it working on the latest update? I keep rolling back as I have three integrations not working with the latest update

**Comment by bobslaede** [comment]
Latest update seems to work. Had to redo one node from HA that. A media_player that when off doesn't return volume level anymore. Rest seems to work.

**Comment by shawwellpete** [comment]
Very odd, I have upgraded HA to core-2021.12.4 and fixed a couple of other issues. Node-red still the same, no status or debug output. Restarting Node red now makes no difference either.

This is a weird one, seems to be a rare problem

**Comment by shawwellpete** [comment]
Update in case any one else is having the same problem...

I did find a solution, or rather what was causing the problem. When you import flows from the web from examples you end up with more than one home assistant server, I ignored the extra ones. It didn't cause any problems for a while until this happened but deleting the rogue servers from here fixed the issue.
![2022-01-29_12-20-11](https://user-images.githubusercontent.com/74064242/151660859-65c8f307-f4ef-42bf-86dd-f598ab065e08.jpg)
